user/pages/05.configuration-tools/10.workflow/14.add_notification/item.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'How to send notification to users inside the CRM'
+metadata:
+    description: 'How to use Add notification Workflow task to send notifications about specific tasks to different users inside the CRM.'
+    author: 'Enea Dudija'
+content:
+    items:
+        - '@self.children'
+    limit: 5
+    order:
+        by: date
+        dir: desc
+    pagination: true
+    url_taxonomy_filters: true
+taxonomy:
+    category:
+        - workflow
+    tag:
+        - howto
+---
+
+
+
+**Add Notification**
+
+  The add notification Workflow task helps us build an automation that notifies the user inside the CRM through a popup window for specific eminders and pending tasks according to the conditions we set.
+  These notifications will appear in the notification panel and Pending Action widget.
+------------------------------
```

Comment:
convert this to three equal signs: `===`  gravcms understands that the three equal signs separate the content into a sub-description of the page and will use it when returning search results and some other summarys